cmd/nginx-ingress/main.go

```diff
@@ -195,7 +196,7 @@ func main() {
 	}
 
 	if *versionFlag {
-		fmt.Printf("Version=%v GitCommit=%v\n", version, gitCommit)
+		fmt.Printf("Version=%v GitCommit=%v Date=%v\n", version, commit, date)
```

Comment:
do we need to update the doc for the version flag?

Reply:
I've updated the flag description, is that it?

Reply:
looks good. thanks!
we have the flag docs here as well though https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/